YCloudYUSA/yusaopeny_gated_content
# fix: DS-1240 Fix recurring event description

**Related Issue/Ticket:**

**PLEASE CHECK BASE BRANCH FOR YOUR PR**
**ONLY urgent and approved fixes for point release should go to master branch**

https://yusa.atlassian.net/browse/DS-1240

## Steps to test:

Empty description:

- Add a Live Stream or Virtual Event with no description
- Try to load the page

Series description:

- Add a description to the event series
- the description should show and the page should load

Instance description:

- Override the description on one instance
- the overridden description should show and the page should load

## Quality checks:

Please check these boxes to confirm this PR covers the following cases:

- Maintaining our upgrade path is essential. Check one or the other:
- Front end fixes should be tested against all of the Open Y Themes.
  team in Slack
